_posts/bmb4227/2020-09-02-first-reflection-bmb4227.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+author: bmb4227
+title: "Brittany's first reflection!"
+---
+For the most part, things are making sense to me. It just feels like I need a lot of practice and everything is very trial and error for me at this point. I appreciate your note in the why python reading that you will give less precise information but more useful. I think that that is useful to the way I learn personally. I’m ok not knowing everything up front because it overwhelms me. When I understand the basics of smaller pieces it makes understanding the whole more easily. I really enjoyed the variables exercises. It felt a lot like doing a puzzle which I really like. The most difficult part for me was figuring out the second problem, multiplying the hours and the rate to get the pay. I did not realize at first that in the problem I needed to multiply int(hours) by int(rate). In an example in Ch. 2 Variables you had put:
+>>> prompt = 'What...is the airspeed velocity of an unladen swallow?\n'
+>>> speed = input(prompt)
+What...is the airspeed velocity of an unladen swallow?
+17
+>>> int(speed)
+17
+>>> int(speed) + 5
+22
```

Comment:
This is great that you've included code in your answer.

We haven't gotten there yet, but if you'd like things like this to display like this:
```
>>> prompt = 'What...is the airspeed velocity of an unladen swallow?\n'
>>> speed = input(prompt)
What...is the airspeed velocity of an unladen swallow?
17
>>> int(speed)
17
>>> int(speed) + 5
22
```
put three backticks (\`) above and below them. This is a convenient feature of Markdown, which both our blog and these Github comments support, for showing code in a different way than text.